js-core/homeworks/homework-6/index.html

```diff
@@ -2,10 +2,10 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Home work 1</title>
```

Comment:
why you move index from first homework to 6 ? and than rename it

Reply:
Because I made a new branch from master, and master branch has those files with first homework. Maybe I do something wrong?